src/sparse/csmat.rs

```diff
@@ -249,6 +328,42 @@ where
             }),
         }
     }
+
+    /// Try create an owned `CSR` matrix from moved data.
```

Comment:
Actually it's also possible to create a `CsMatViewMut` using this constructor I think.

Reply:
I've removed all mentions of `owned` and made some examples on `new` which shows how one can construct different type (aliases) directly